files/project_team/IFCTunnelDep2.2.ifc

```diff
@@ -0,0 +1,630 @@
+#42= IFCAXIS2PLACEMENT3D(#43,#44,#46);
+#43= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#44= IFCDIRECTION((0.0,0.0,1.0));
+#45= IFCDIRECTION((1.0,0.0,0.0));
+#46= IFCDIRECTION((1.0,0.0,0.0));
+#47= IFCDIRECTION((0.0,1.0));
+#48= IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE.,#49,IFCNONNEGATIVELENGTHMEASURE(0.0),IFCNONNEGATIVELENGTHMEASURE(1508.3410985277),#52);
+#49= IFCAXIS2PLACEMENT2D(#50,#51);
+#50= IFCCARTESIANPOINT((162572.948873,6721939.071596));
+#51= IFCDIRECTION((-0.99484944,-0.10136364));
+#52= IFCLINE(#53,#54);
+#53= IFCCARTESIANPOINT((0.0,0.0));
+#54= IFCVECTOR(#55,1.0);
+#55= IFCDIRECTION((-0.99484944,-0.10136364));
+#56= IFCALIGNMENTHORIZONTALSEGMENT(', code=POB',', code=TS',#57,-3.04005462523801,0.0,0.0,1508.341099,$,.LINE.);
+#57= IFCCARTESIANPOINT((162572.948873,6721939.071596));
+#58= IFCALIGNMENTSEGMENT('3f3U42JTHDjAX1u4qJChx$',$,'E',$,$,#59,#62,#72);
+#59= IFCLOCALPLACEMENT($,#60);
+#60= IFCAXIS2PLACEMENT3D(#61,$,$);
+#61= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#62= IFCPRODUCTDEFINITIONSHAPE($,$,(#63));
+#63= IFCSHAPEREPRESENTATION(#41,'Axis','Segment',(#64));
+#64= IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE.,#65,IFCPARAMETERVALUE(0.0),IFCNONNEGATIVELENGTHMEASURE(92.0),#68);
+#65= IFCAXIS2PLACEMENT2D(#66,#67);
+#66= IFCCARTESIANPOINT((161072.376573,6721786.180648));
+#67= IFCDIRECTION((-0.99484944,-0.10136365));
+#68= IFCCLOTHOID(#69,418.090899207337);
+#69= IFCAXIS2PLACEMENT2D(#70,#71);
+#70= IFCCARTESIANPOINT((0.0,0.0));
+#71= IFCDIRECTION((-0.99484944,-0.10136365));
+#72= IFCALIGNMENTHORIZONTALSEGMENT(', code=TS',', code=SC',#73,-3.04005461353714,0.0,1900.0,92.0,$,.CLOTHOID.);
+#73= IFCCARTESIANPOINT((161072.376573,6721786.180648));
+#74= IFCALIGNMENTSEGMENT('2jdhB1U8DCIuP8GRLWMeWX',$,'E',$,$,#75,#78,#88);
+#75= IFCLOCALPLACEMENT($,#76);
+#76= IFCAXIS2PLACEMENT3D(#77,$,$);
+#77= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#78= IFCPRODUCTDEFINITIONSHAPE($,$,(#79));
+#79= IFCSHAPEREPRESENTATION(#41,'Axis','Segment',(#80));
+#80= IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE.,#81,IFCNONNEGATIVELENGTHMEASURE(0.0),IFCPARAMETERVALUE(0.0213755629473684),#84);
+#81= IFCAXIS2PLACEMENT2D(#82,#83);
+#82= IFCCARTESIANPOINT((160980.931044,6721776.117138));
+#83= IFCDIRECTION((-0.99210406,-0.12541741));
+#84= IFCCIRCLE(#85,1900.0);
+#85= IFCAXIS2PLACEMENT2D(#86,#87);
+#86= IFCCARTESIANPOINT((238.293085,-1884.99772));
+#87= IFCDIRECTION((-0.12541741,0.99210406));
+#88= IFCALIGNMENTHORIZONTALSEGMENT(', code=SC',', code=CS',#89,-3.01584409831919,1900.0,1900.0,40.61357,$,.CIRCULARARC.);
+#89= IFCCARTESIANPOINT((160980.931044,6721776.117138));
+#90= IFCALIGNMENTSEGMENT('0x4tMtFkf4ZedqVI8bIgk9',$,'E',$,$,#91,#94,#104);
+#91= IFCLOCALPLACEMENT($,#92);
+#92= IFCAXIS2PLACEMENT3D(#93,$,$);
+#93= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#94= IFCPRODUCTDEFINITIONSHAPE($,$,(#95));
+#95= IFCSHAPEREPRESENTATION(#41,'Axis','Segment',(#96));
+#96= IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE.,#97,IFCPARAMETERVALUE(-92.0),IFCNONNEGATIVELENGTHMEASURE(92.0),#100);
```

Comment:
@SergejMuhic I can't remember: what was the conclusion on mixing `IfcParameterValue` with `IfcNonNegativeLengthMeasure`?

Reply:
> @SergejMuhic I can't remember: what was the conclusion on mixing `IfcParameterValue` with `IfcNonNegativeLengthMeasure`?

Don't know? @SergejMuhic ?

Reply:
Implementer agreement, sort of.